packages/protocol/test/governance/epochrewards.ts

```diff
@@ -91,6 +91,7 @@ contract('EpochRewards', (accounts: string[]) => {
 
```

Comment:
Freezing should be tested in this file

Reply:
The function whose behavior is modified (`updateTargetVotingYield`) can only be called from within geth (`require(msg.sender == address(0))`), so from these ganache-based tests we won't be able to observe a change in behavior. This is tested in the e2e tests instead.

Reply:
Ah ok.